src/nvim/ex_docmd.c

```diff
@@ -575,14 +577,20 @@ int do_cmdline(char *cmdline, LineGetter fgetline, void *cookie, int flags)
     if (next_cmdline == NULL) {
       // Need to set msg_didout for the first line after an ":if",
       // otherwise the ":if" will be overwritten.
-      if (count == 1 && getline_equal(fgetline, cookie, getexline)) {
-        msg_didout = true;
-      }
-      if (fgetline == NULL
-          || (next_cmdline = fgetline(':', cookie,
-                                      cstack.cs_idx <
-                                      0 ? 0 : (cstack.cs_idx + 1) * 2,
-                                      true)) == NULL) {
+      int indent = 0;
+      if (getline_equal(fgetline, cookie, getexline)) {
+        if (count >= 1) {
+          indent = cstack.cs_idx < 0 ? 0 : (cstack.cs_idx + 1) * 2;
```

Comment:
Why not move the assignment of `indent` out of the condition?
```suggestion
      int indent = cstack.cs_idx < 0 ? 0 : (cstack.cs_idx + 1) * 2;
      if (getline_equal(fgetline, cookie, getexline)) {
        if (count >= 1) {
```

Reply:
Did it to indicate that `indent` isn't used when `getline_equal(getexline) == false`. But either works for me.